src/app/components/Card/Card.tsx

```diff
@@ -41,6 +37,10 @@ const Container = styled.article`
   border-radius: 0.5rem;
   width: 90%;
   margin: auto;
+  padding-bottom: 0.5rem;
+  & + & {
+    margin-top: 1rem;
+  }
```

Comment:
I haven't seen this yet - what does it do? Would be happy to know more! :)

Reply:
It does make sure that the margin-top is only used when a Container is next to another Container (to be more precise here: if two are on top of each other).
Styled components documentation calls is "<Thing> next to <Thing>"
[See here the third example in the box](https://styled-components.com/docs/basics#pseudoelements-pseudoselectors-and-nesting)